addons/mrp/mrp.py

```diff
@@ -155,6 +155,12 @@ class mrp_bom(osv.osv):
     _description = 'Bill of Material'
     _inherit = ['mail.thread']
 
+    def _get_bom_ids(self, cr, uid, ids, context=None):
+        result = {}
+        for bom in self.pool.get('product.template').browse(cr, uid, ids, context=context).bom_ids:
```

Comment:
What will happen if browse returns more than one browse record? We take only the bom_ids of the first one? That's probably incorrect